dev/test

```diff
@@ -27,6 +27,9 @@ cd "${REPO_ROOT}"
 
 set -x
 
+# Download the kubebuilder assets for envtest
+export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
```

Comment:
As of https://github.com/kubernetes-sigs/controller-runtime/pull/3135 this isn't really strictly necessary anymore. Would be worthwhile getting rid of while we're doing an overhaul, imo!

Reply:
OK that is very cool.  I don't think it is released yet on a controller-runtime tag (at least not that I could see).  We could keep our main/master branch close to the upstream controller-runtime main/master branch, now that we have branches!